### PR TITLE
Go: use vendor or dep tool during builds

### PR DESCRIPTION
**Pull request by denismakogon**
Closes: #141

### CODE REVIEW

File: langs/go.go [hunk 1]
```diff
@@ -35,17 +35,20 @@ func (lh *GoLangHelper) RunFromImage() (string, error) {
...
+	if !vendor {
+		if exists("Gopkg.toml"){
+			r = append(r, "RUN go get -u github.com/golang/dep/cmd/dep",
+				"RUN cd /go/src/func/ && dep ensure -v")
+		}
+	}
+
 	r = append(r, "RUN cd /go/src/func/ && go build -o func")
+	// get rid of vendor after binary executable creation
+	r = append(r, "RUN cd /go/src/func/ && rm -fr vendor/")
```
Comment (treeder) [inline langs/go.go]: This isn't necessary in multi-stage.

Comment (denismakogon) [reply langs/go.go]: done

**Comment by denismakogon** [comment]
Okay, i merged PR for fdk-go to get the example app from there, starting this moment CI should be green (at least locally it's happy with go func tests).

I feel a bit bad about not including glide to this PR, but it requires to include glide tool to the `fnproject/go:dev`. Will open PR for that.

**Comment by treeder** [comment]
Don't worry about glide, dep seems to be taking it's place.

**Comment by denismakogon** [comment]
Wish kubernets knew that =/

**Pull request merged**